gradle/libs.versions.toml

```diff
@@ -0,0 +1,23 @@
+[versions]
+gradle = "8.0.0"
```

Comment:
Same comment as in the previous PR, this is android-gradle-tools and 8.2.0 is latest